### DISCUSSION TITLE
Events content

### DISCUSSION THREAD

**Discussion by chimp1984** [bisq-network/bisq2#363] [General]
Here is a collection of ideas how we could use the events area in Bisq 2:

## Goals:
- Create a **pulse**, so people open up the app to see whats going on. Frequent updates required (min 1-2 week)
- Let people **connect** to work together and become active (create a community,...)
- Promote like minded projects

## Name
Depending on the selected content we might need to look for other terms as 'Events' might be too narrow.
Alternative names:
- Connect
- Pulse
- Whats up
- Activities

### Events
- Trade events ("Flash mob trading") to get better liquidity
- Upcoming Satoshi square events
- Upcoming Bitcoin Meetups (search by location if there are too many entries)
- Upcoming conferences
- Promote the next shitcoin airdrop.  No, just joking... :-)

### Podcasts
- Upcoming podcasts
- Past podcasts

### Projects
- No KYC services
- Nodes projects
- Bitcoin only services
- Bitcoin Beach

### Wanna become active?
- How to become a Bisq contributor 
- How to orangize a Satoshi square event (face to face trading, casual meeting without need for speaker)
- How to start a local BTC community (Bitcoin beach guide)

## Content management
We do not want to implement a content management system, so we need to find a way to limit effort for dynamic content.
One way could be that users can add content based on a few categories (confernce, meetup,...) with the min. required data (title, location, date, link) and that request gets sent to moderators who review it and if all is ok, they confirm and publish it. The events will have a max. time to life (e.g. 1 month) and after the event date they get removed. 

We could add a rating tool to the items so users can leave feedback (maybe even text feedback next to star rating)?

The 'Wanna become active' area is static content and could maybe also be moved to the 'Learn' section.

The 'Projects' area could be hard coded and updated with sourc code at releases, as its not expected that they change often. Maybe we could add a promoted project each release cycle?

Maybe thats already too much... just brain stormed what could fit into it...

**Comment by chimp1984** [comment]
Actually using an additional chat section for events migth be a good idea. Moderator rules can be more strict there.

**Comment by Conza88** [comment]
_Promote the next shitcoin airdrop. No, just joking... :-)

Projects
Bitcoin only services_

👀
